api-catalog-ui/frontend/src/components/Wizard/configs/wizard_base_categories.jsx

```diff
@@ -42,6 +45,7 @@ export const baseCategories = [
                 value: '',
                 question: 'The base URL of the service (the consistent part of the web address):',
                 validUrl: true,
+                tooltip: 'https://${samplehost}: ${sampleport}/${sampleservice}',
```

Comment:
There is a space between `${samplehost}:` and `${sampleport}`

Reply:
I added the spaces so the tooltip text wouldn't overflow (tooltip has a maximum width). I will see if there's a way to break the lines automatically.